README.md

```diff
@@ -142,6 +143,71 @@ invalid scope:
 ;; {:status 401, :body {:message "missing required scope", :required :my-scope, :scopes (:not-my-scope)}}
 ```
 
+### Support middleware `keyset` refresh
```

Comment:
"Support middleware `keyset` refresh" => "`keyset` refresh"
it already lives under the "`use in middleware`" section